ssg/templates.py

```diff
@@ -120,66 +132,26 @@ def __init__(
                 output_dir = self.remediations_dir
             dir_ = os.path.join(output_dir, lang_dir)
             self.output_dirs[lang_name] = dir_
-        # scan directory structure and dynamically create list of templates
+        # Scan directory structure and dynamically create list of templates
```

Comment:
While you are at refactoring, would it be possible to extract mapping of output directories and mapping of templates into private methods? If you give them correct names, you don't need comments any more.

Reply:
NP.